server/controllers/match/placementRequests/spaceBookingsController.test.ts

```diff
@@ -72,7 +76,8 @@ describe('SpaceBookingsController', () => {
         errorSummary: {},
         errors: [],
         dates: placementDates(startDate, durationDays),
-        essentialCharacteristics: filterOutAPTypes(placementRequestDetail.essentialCriteria),
+        // essentialCharacteristics: filterOutAPTypes(placementRequestDetail.essentialCriteria),
+        essentialCharacteristics: filterOutAPTypes(criteria),
```

Comment:
Not sure we should need to filter out AP types, if we know `criteria` is already of the `Cas1SpaceBookingCharacteristic` subset

Reply:
Yes, the type conversion is the filter so it doesn't need doing twice. Again, I switched for `filterToSpaceBookingCharacteristics` to reduce to only the set of criteria we want to save in the space booking.